pkg/core/blockchain.go

```diff
@@ -276,6 +291,9 @@ func (bc *Blockchain) Run() {
 		close(bc.runToExitCh)
 	}()
 	go bc.notificationDispatcher()
+	if orc := bc.contracts.Oracle.Module; orc != nil {
+		go bc.contracts.Oracle.Module.(*oracle.Oracle).Run()
```

Comment:
I think it can be done outside of the `Blockchain`.

Reply:
How? I don't want to expose `native.Oracle` from blockchain.

Reply:
Fixed.